SCR/Pipfile

```diff
@@ -6,7 +6,7 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-integration-adaptors-common = {editable = true,path = "./../common"}
+integration-adaptors-common = {editable = true,git = "https://git@github.com/nhsconnect/integration-adaptor-common.git",tag = '0.1.6'}
```

Comment:
tag = '0.1.6' for every project, maybe it should be a global value.

Reply:
this will be fixed in next PR where there is a single Pipfile for the whole MHS